Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalHBHEMuonFilter_Output_cff.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
+
+# output block for alcastream HCAL HBHEMuon
+# output module 
+#  module alcastreamHcalHBHEMuonOutput = PoolOutputModule
+OutALCARECOHcalCalHBHEMuonFilter_noDrop = cms.PSet(
+    # use this in case of filter available
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOHcalHBHEMuonFilter')
+        ),
+    outputCommands = cms.untracked.vstring( 
+        'keep *_hbhereco_*_*',
+        'keep *_ecalRecHit_*_*',
+        'keep *_offlineBeamSpot_*_*',
+        'keep *_hltTriggerSummaryAOD_*_*',
+        'keep *_TriggerResults_*_*',
+        'keep *_generalTracks_*_*',
```

Comment:
maybe its better to remove the event content from this PR so that the resources can then be discussed when they are more appropriately defined? [as now its basically RECO]